### PR TITLE
importer: provide warning for combined exchange reactions

### PR DESCRIPTION
**Pull request by spikeliu**
reference: #1

### CODE REVIEW

**Comment by jonls** [comment]
The check is good but I think it should go in the `model_medium()` function which is the function that actually produces the medium definition.

File: psamm_import/importer.py [hunk 1]
```diff
@@ -225,6 +225,12 @@ def model_medium(model, default_flux_limit):
             continue
 
         if len(equation.compounds) != 1:
+            # Provide warning for exchanges reactions with more than
+            # one compound, they won't be put into the medium definition
+            if (len(equation.left) == 0) != (len(equation.right) == 0):
+                logger.warning('Exchange reaction {} has more than one'
+                               ' compound, so it won\'t be converted to'
+                               ' medium format'.format(reaction.id))
```
Comment (jonls) [inline psamm_import/importer.py]: - In your comment: "exchanges reactions" -> "exchange reactions"
- You have some excess `(` and `)` in the `if` statement
- I would change the message to something like: "Exchange reaction {} has more than one compound; it was not converted to medium compounds"

Comment (spikeliu) [inline psamm_import/importer.py]: I think no excess ().

Comment (jonls) [inline psamm_import/importer.py]: Yes, you are right 👍

**Pull request merged**